_episodes_rmd/01-introduction.Rmd

```diff
@@ -116,11 +116,7 @@ server:
 <img src="../fig/rstudio_login_screen.png" alt="rstudio default session" style="width: 1000px;"/>
 
 Enter your user credentials and click <kbd>Sign In</kbd>. The credentials for
-the genomics Data Carpentry instances are:
-
- > **username**: dcuser
- >
- > **password**: data4Carp
+the genomics will be provided by your instructors. 
```

Comment:
Add back in 'Data Carpentry instances'?

Reply:
Nice catch @naupaka. Thanks for the careful read through. I'll merge this as soon as the checks pass.